### PR TITLE
Fixing some typos that Jeroen52 didn't notice.

### PR DESCRIPTION
**Pull request by BestName**
_cough_ #9331 _cough_

### CODE REVIEW

**Comment by Probe1** [comment]
at this rate we will proof check the whole repo.

**Comment by SysadminJeroen** [comment]
@Probe1 And that is a problem how?

**Comment by Probe1** [comment]
We may be expected to put in an honest days work

**Comment by xpcybic** [comment]
I almost considered doing exactly that until I realized the sheer magnitude of what I was getting myself into and came to my senses

**Comment by Intigracy** [comment]
You guys can have yours merged after mine is merged, just to see if there's conflicts.

If there are, that means I fixed it before you did.

Rude people smh.

**Comment by clusterfack** [comment]
Conflicts :^)

**Comment by BestName** [comment]
`¯\_(ツ)_/¯`

**Comment by SonixApache** [comment]
>monstrocity

File: code/game/objects/items/mountable_frames/paintings.dm [hunk 1]
```diff
@@ -242,7 +242,7 @@ var/global/list/available_paintings = list(
 			desc = "A painting depicting a compact but efficient hospital. The red really helps make the normally drab pallet pop."
 		if("prophecy")
 			name = "\improper Prophetic Mural"
-			desc = "A copy of an ancient mural. It depicts a blue armored warrior fighting off an inhuman monstrocity. Its said many still wait for its conclusion."
```
Comment (SonixApache) [inline code/game/objects/items/mountable_frames/paintings.dm:245]: > Monstrocity

Comment (SysadminJeroen) [inline code/game/objects/items/mountable_frames/paintings.dm:245]: monstrocity

**Pull request closed without merging**